packages/ckeditor5-language/src/utils.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/utils
+ */
+
+import { getLanguageDirection } from '@ckeditor/ckeditor5-utils/src/language';
```

Comment:
This import is incorrect and I wonder why it wasn't caught by the linter. It should go through `ckeditor5/...`.